gna_device.c

```diff
@@ -190,7 +190,6 @@ static const struct drm_driver gna_drm_driver = {
 
 	.name = DRIVER_NAME,
 	.desc = DRIVER_DESC,
-	.date = DRIVER_DATE,
```

Comment:
When I merge this, I'll also remove the DRIVER_DATE macro from the `gna_device.h`.